### PR TITLE
all products vertical slice

### PR DESCRIPTION
**Pull request by CMorgan-Feir**
### Assignee Tasks

* [ ] check for consistency between approaches
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: client/components/all-products.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+// import SingleProduct from './SingleProduct'
```
Comment (rrossman25) [inline client/components/all-products.js]: UNCOMMENT!

File: client/store/all-products.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      dispatch(gotProducts(data))
+    } catch (error) {
+      console.log('Something went wrong: ', error)
+    }
+  }
+}
+
+const initialState = {
+  products: [],
+  product: {}
```
Comment (rrossman25) [inline client/store/all-products.js]: ?

File: server/api/all-products.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router()
+const {AllProducts} = require('../db/models')
```
Comment (dotsalot) [inline server/api/all-products.js]: AllProducts  is just Product

File: client/store/all-products.js (hunk 2 above)
Comment (rrossman25) [reply client/store/all-products.js]: Why do you have both products and product?

**Pull request merged**